contracts/wrapper.md

```diff
@@ -49,3 +49,5 @@ function swap(
 | `SIGNATURE_MUST_BE_SENT`          | The signature field is blank. This is not allowed on the Wrapper.          |
 | `VALUE_MUST_BE_SENT`              | No ETH was provided for an order that was expecting ETH to wrap.           |
 | `VALUE_MUST_BE_ZERO`              | ETH was provided for an order that was not expecting ETH to wrap.          |
+| `ETH_RETURN_FAILED`               | Unwrapping weth to ether failed.                             |
```

Comment:
This is more `transferring ETH to the recipient failed`

Reply:
and capitalise WETH

Reply:
I realise now the capitalisation of WETH is irrelevant after the description changes aha